.circleci/config.yml

```diff
@@ -60,12 +60,19 @@ commands:
           rust-version: <<parameters.rust-version>>
       - run:
           name: Test
-          command: cargo test --all --verbose -- --nocapture
+          command: GLEAN_TEST_COVERAGE=$(realpath glean_coverage.txt) cargo test --all --verbose -- --nocapture
       - run:
           name: Test Glean with rkv safe-mode
           command: |
             cd glean-core
             cargo test -p glean-core --features rkv-safe-mode -- --nocapture
+      - run:
+          name: Upload coverage report
+          command: |
+            sudo apt install python3-pip
+            pip3 install git+https://github.com/mdboom/glean_parser.git@coverage-analysis
```

Comment:
This just installs a branch of glean_parser.  No longer necessary once that branch is merged and we make a release.